src/delegation/DelegationRootNode.ts

```diff
@@ -21,7 +21,9 @@ export default class DelegationRootNode extends DelegationBaseNode
    * @param delegationId unique identifier of the delegation root
    * @returns promise containing [[DelegationRootNode]] or [undefined]
    */
-  public static async query(delegationId: string) {
+  public static async query(
+    delegationId: string
+  ): Promise<DelegationRootNode | undefined> {
```

Comment:
Same question regarding `undefined` vs `null`

Reply:
we should switch to null

Reply:
https://github.com/KILTprotocol/sdk-js/issues/153